extensions/data-workspace/src/dialogs/utils.ts

```diff
@@ -0,0 +1,104 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
VS only ran on Windows so there was no concern of cross-platform compatibility.  Now that we run on Unix and Windows, do we want to block or warn when choosing a name that wouldn't work on the other platform?

Reply:
In the changes, it does take into consideration for invalid file chars in Windows and Unix (line 12). So it does character validation based on the platform.